### ISSUE TITLE
data set problem

### ISSUE THREAD

**Issue by bianbianbian1202** [microsoft/nni#3196]
hello！How do I train my data set in DARts? Or where should you change the code?

**Comment by ultmaster** [comment]
You can modify the code to use your own dataset. Specifically, replace the CIFAR10 part with your own torch dataset.

**Comment by bianbianbian1202** [comment]
Thanks for your reply.
Sorry, this is the first time for me to use such a framework. Could you please tell me more about the modifications? Such as:



dataset_train, dataset_valid = datasets.get_dataset("cifar10")



datasets.py

if cls == "cifar10":

dataset_train = CIFAR10(root="./data", train=True, download=True, transform=train_transform)

dataset_valid = CIFAR10(root="./data", train=False, download=True, transform=valid_transform)

**Comment by R1ckShi** [comment]
If your dataset in simple as CIFAR (only x and y), you can just define your torch dataloader for train and valid, and send them to `Trainer()`. 
But in most situations, you need to customize your own trainer and define the data sampling, metrics, accuracy, and so on.
